src/BingAIClient.js

```diff
@@ -64,11 +96,13 @@ export default class BingAIClient {
                 'sec-fetch-site': 'same-origin',
                 'x-ms-client-request-id': crypto.randomUUID(),
                 'x-ms-useragent': 'azsdk-js-api-client-factory/1.0.0-beta.1 core-rest-pipeline/1.10.0 OS/Win32',
-                cookie: this.options.cookies || (this.options.userToken ? `_U=${this.options.userToken}` : undefined),
+                'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.50',
+                cookie: this.options.cookies || `_U=${this.options.userToken}`,
```

Comment:
You'll probably see this when fixing merge conflicts, as I've made the change there, but the cookie option needs to be able to handle the cookie not being set at all now.

Reply:
I will take the changes back in PR.